builder/hyperv/common/ssh_config.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"time"
+
+	"github.com/mitchellh/packer/helper/communicator"
+	"github.com/mitchellh/packer/template/interpolate"
+)
+
+type SSHConfig struct {
+	Comm communicator.Config `mapstructure:",squash"`
+
+	// These are deprecated, but we keep them around for BC
+	// TODO(@mitchellh): remove
+	SSHKeyPath     string        `mapstructure:"ssh_key_path"`
+	SSHWaitTimeout time.Duration `mapstructure:"ssh_wait_timeout"`
```

Comment:
These should be dropped, see 
https://github.com/mitchellh/packer/blob/master/fix/fixer_sshkeypath.go